apps/better-ccusage/src/opencode-adapter.ts

```
@@ -0,0 +1,455 @@
+			const db = new DatabaseSync(dbPath);
+			db.exec(`
+				CREATE TABLE message (id TEXT PRIMARY KEY, session_id TEXT, time_created INTEGER, time_updated INTEGER, data TEXT);
+				CREATE TABLE session (id TEXT PRIMARY KEY, directory TEXT);
+			`);
+			const insertMessage = db.prepare(
+				'INSERT INTO message (id, session_id, time_created, time_updated, data) VALUES (?, ?, ?, ?, ?)',
+			);
+			const insertSession = db.prepare('INSERT INTO session (id, directory) VALUES (?, ?)');
+			insertSession.run('ses_1', '/home/user/projects/app');
+			insertMessage.run(
+				'msg_1', 'ses_1', 1777388092686, 1777388092686,
```

ESLint

[error] Should have line breaks between items, in node CallExpression

(antfu/consistent-list-newline)